py/autoevals/ragas.py

```diff
@@ -894,7 +903,7 @@ class AnswerSimilarity(OpenAILLMScorer):
     Measures the similarity between the generated answer and the expected answer.
     """
 
-    def __init__(self, pairwise_scorer=None, model=DEFAULT_RAGAS_MODEL, **kwargs):
+    def __init__(self, pairwise_scorer=None, model=DEFAULT_RAGAS_EMBEDDING_MODEL, **kwargs):
```

Comment:
note that updated the default model here. with the code as it was, it would have used the chat model